<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>工具</title>
    <style>
         body{
            margin: 0;
            background-color:#c1dac1;
            }
         .p1{
            height: 150px;
            width: 150px;
            margin-left: 45%;
            }
            .t1{
            width: 2.1%;
            }
        .t2{
            width: 3.4%;
            color: white;
            }
        .t1 img{
            width: 25px;
            height: 25px;
            margin-left: 50%;
            }
        li{
            line-height: 3rem;
            font-size: 20px;
        }
        a{
            cursor: pointer;
            text-decoration: none;
        
        }
        .a1 {
            font-size: 22px;
            float: left;
            display: block;
            color: #f2f2f2;
            text-align: center;
            text-decoration: none;
            }
        h3{
            font-size: 30px;
            }
        .p2{
            font-size: 20px;
        }
        .footer {
            margin-top: 4%;
            text-align: center;
            background-color:#2c6d36;
            color: white;
            height: 110px;
            }
    </style>
</head>

<table style="background-color:#9ac09a; width: 100%;">
    <tr>
        <td><img src="../picture/fq2.png" style=" height: 210px; width: 600px; margin-left: 2%;"></td>
        <td><img src="../picture/lab-logo.png" style=" height: 210px;width: 240px;"></td>
        <td><img src="../picture/fq1.png" alt="" style=" height: 210px; width: 610px; float: right; margin-right: 6%;"></td>
    </tr>
</table>


<body>
    <table style="width: 100%;  height: 50px; background-color:#2c6d36">
        <tr>
            <td class="t1"><img src="../picture/首页-01.png" alt=""></td>
            <td class="t2"><a href="../首页/首页.html" class="a1">首页</a></td>

            <td class="t1"><img src="../picture/项目成员.png" alt=""></td>
            <td class="t2"><a href="../内容/人员.html" class="a1">成员介绍</a></td>

            <td class="t1"><img src="../picture/层面方向.png" alt=""></td>
            <td class="t2"><a href="../内容/研究.html" class="a1">研究方向</a></td>

            <td class="t1"><img src="../picture/活动，日历.png" alt=""></td>
            <td class="t2"><a href="../内容/活动.html" class="a1">活动交流</a></td>

            <td class="t1"><img src="../picture/培训.png" alt=""></td>
            <td class="t2"><a href="../内容/课题.html" class="a1">培训课题</a></td>

            <td class="t1"><img src="../picture/工具3.png" alt=""></td>
            <td class="t2"><a href="../内容/工具.html" class="a1">在线工具</a></td>

            <td class="t1"><img src="../picture/加入书架.png" alt=""></td>
            <td class="t2"><a href="../内容/招聘.html" class="a1">加入我们</a></td>
        </tr>
    </table>


<table style="width: 1450px;">
    <tr>
        <td style="float: left; margin-left: 8%;line-height: 3rem; font-size: 20px;margin-top: 5%; ">
            <table border="2" style="width: 350px; border-style: solid; border-color: #000;">
                <tr><td style="border: none;"><a href="#0"  style="color: black;">1. 数据库</a></td></tr>
                <tr><td style="border: none;"><a href="#1"  style="color: black;">2. Pairwise sequence alignment</a></td></tr>
                <tr><td style="border: none;"><a href="#2"  style="color: black;">3. Multiple sequence alignment</a></td></tr>
                <tr><td style="border: none;"><a href="#3"  style="color: black;">4. Sequence similarity search</a></td></tr>
                <tr><td style="border: none;"><a href="#4"  style="color: black;">5. Web-based bioinformatics platforms</a></td></tr>
                <tr><td style="border: none;"><a href="#5"  style="color: black;">6. Bioinformatics packages to be downloaded and installed locally</a></td></tr>
                <tr><td style="border: none;"><a href="#6"  style="color: black;">7. Protein sequence analysis and function prediction</a></td></tr>
                <tr><td style="border: none;"><a href="#7"  style="color: black;">8. Motif finding</a></td></tr>
                <tr><td style="border: none;"><a href="#8"  style="color: black;">9. Primer Design</a></td></tr>
                <tr><td style="border: none;"><a href="#9"  style="color: black;">10. Gene identification</a></td></tr>
                <tr><td style="border: none;"><a href="#10"  style="color: black;">11. Molecular phylogeny</a></td></tr>
                <tr><td style="border: none;"><a href="#11"  style="color: black;">12. 3D visualization</a></td></tr>
                <tr><td style="border: none;"><a href="#12"  style="color: black;">13. Protein modeling</a></td></tr>
                <tr><td style="border: none;"><a href="#13"  style="color: black;">14. RNA tools</a></td></tr>
                <tr><td style="border: none;"><a href="#14"  style="color: black;">15. List of bioinformatics tools at international bioinformatics centers</a></td></tr>
            </table>
        </td>
        
        <td style="float: right;margin-right:15%; "> 
        <table style="width: 700px;">
            <tr>
                <td>
                    <h3>常用工具</h3>
                    <ol>
                        <li id="0">数据库：<a href="http://202.112.170.207">Solanaceae Genomics Database </a></li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/clustalo/"> Clustal Omega</a>Clustal Omega is a new multiple sequence alignment program that uses seeded guide trees and HMM profile-profile techniques to generate alignments.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/muscle/"> Muscle</a> - Multiple sequence comparison with better accuracy and higher speed.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/mafft/"> MAFAT</a> - High speed multiple sequence alignment using fast fourier transformation, at EBI.</li>
                        <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi"> NCBI BLAST</a> - The central point of the NCBI BLAST server which provides whole functionality of DNA and protein sequence database search programs including PSI-BLAST and PHI-BLAST, and the whole list of databases in different scales and different genomes.</li>
                        <li><a href="https://asia.ensembl.org/Multi/Tools/Blast"> Ensembl BLAST/BLAT</a> - The BLAST and BLAT search tool implemented in the Ensembl platform.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/hmmer/"> HMMER Search</a>- Sequence profile Hidden Markov Model (HMMER) server developed and maintained by EBI.</li>
                        <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi"> NCBI BLAST tools</a> - The BLAST platform at NCBI with many tools including Needleman-Wunsch global pairwise sequence alignment and multiple sequence alignment.</li>
                        <li><a href="http://weblab.gao-lab.org/"> WebLab</a> - The Sequence analysis platform developed by the Center for Bioinformatics (CBI), Peking University.</li>
                        <li><a href="http://hmmer.org/"> HMMER</a>- Profile hidden Markov models to detect conserved domains in protein sequences.</li>
                        <li><a href="https://meme-suite.org/meme/"> MEME</a> - Motif-based sequence analysis tools.</li>
                        <li><a href="http://bioweb.uwlax.edu/genweb/molecular/seq_anal/primer_design/primer_design.htm"> Primer Design</a> - The online primer design tool at BioWeb.</li>
                        <li><a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/"> Primer Blast</a> - NCBI primer design tool with Blast search.</li>
                        <li><a href="https://www.yeastgenome.org/primer3"> Web Primer</a> - Online primer design tool at the yeast genome database.</li>
                        <li><a href="https://www.megasoftware.net/"> MEGA</a> - Molecular Evolutionary Genetics Analysis package.</li>
                        <li><a href="https://evolution.genetics.washington.edu/phylip.html"> Phylip</a> - Phylogeny Inference Package created by Joe Felsenstein at the University of Washington.</li>
                        <li><a href="https://itol.embl.de/"> iTOL</a> - Interactive Tree of Life, for the display and manipulation of phylogenetic trees.</li>
                        <li><a href="https://swissmodel.expasy.org/"> Swiss Model</a> - Automated comparative protein modeling server at SIB.</li>
                        <li><a href="http://www.softberry.com/"> Softberry</a> - Online tools integrated by the Softberry bioinformatics company in US.</li>
                    </ol>
                    <h3>所有工具</h2>
                    <h3 id="1">Pairwise sequence alignment</h3>
                    <ol>
                        <li><a href="https://www.ebi.ac.uk/Tools/psa/emboss_needle/">Needle</a> - Needleman-Wunsch global sequence alignment at EBI.</li>
                        <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">Global Align</a> - Needleman-Wunsch global sequence alignment at NCBI.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/psa/">Stucture Alignment</a> - Pairwise structure and sequence alignment at PDB.</li>
                        <li><a href="">Water</a> - Smith-Waterman local sequence alignment at EBI.</li>
                        <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">Blast2Seq</a> - Align two or more sequences using BLAST at NCBI.</li>
                        <li><a href="">Dotlet</a> - Sequence comparison with dot plot at SIB. Examples</li>
                    </ol>
                    
                    <h3 id="2">Multiple sequence alignment</h3>
                    <ol>
                        <li><a href="https://www.ncbi.nlm.nih.gov/tools/cobalt/">COBALT</a> - NCBI Constraint-based Multiple Alignment Tool.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/clustalo/">Clustal Omega</a>Clustal Omega is a new multiple sequence alignment program that uses seeded guide trees and HMM profile-profile techniques to generate alignments.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/muscle/">Muscle</a> - Multiple sequence comparison with better accuracy and higher speed.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/mafft/">MAFAT</a> - High speed multiple sequence alignment using fast fourier transformation, at EBI.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/kalign/">KALIGN</a> - The web server of fast and accurate multiple sequence alignment at EBI.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/msa/tcoffee/">T-Coffe</a> - Multiple alignments of DNA and protein sequences and structures, at Center for Genomic Regulation, Spain.</li>
                    </ol>
                    
                    <h3 id="3">Sequence similarity search</h3>
                    <ol>
                        <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">NCBI BLAST</a> - The central point of the NCBI BLAST server which provides whole functionality of DNA and protein sequence database search programs including PSI-BLAST and PHI-BLAST, and the whole list of databases in different scales and different genomes.</li>
                        <li><a href="https://ngdc.cncb.ac.cn/blast/home">BIGD BLAST</a> - The BLAST tool at Beijing Institute of Genomics.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/sss/">EBI Sequence Similarity Seacrh</a> - EBI sequence similarity search including FASTA, BLAST and SSEARCH, GGSearch, GLSearch.</li>
                        <li><a href="https://www.sanger.ac.uk/resources/software/blast/">Sanger BLAST</a> - BLAST search services at the Sanger Institute, with special genome databases generated by the completed or ongoing sequencing projects.</li>
                        <li><a href="https://asia.ensembl.org/Multi/Tools/Blast">Ensembl BLAST/BLAT</a> - The BLAST and BLAT search tool implemented in the Ensembl platform.</li>
                        <li><a href="http://genome.ucsc.edu/cgi-bin/hgBlat">UCSC BLAT</a> - The web server for the genome database search, integrated in the UCSC genome browser platform.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/hmmer/">HMMER Search</a>- Sequence profile Hidden Markov Model (HMMER) server developed and maintained by EBI.</li>
                    </ol>
                    
                    <h3 id="4">Web-based bioinformatics platforms</h3>
                    <ol>
                        <li><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">NCBI BLAST tools</a> - The BLAST platform at NCBI with many tools including Needleman-Wunsch global pairwise sequence alignment and multiple sequence alignment.</li>
                        <li><a href="https://www.ebi.ac.uk/Tools/emboss/">EBI EMBOSS tools</a> - The entry page for the EBI EMBOSS tools.</li>
                        <li><a href="https://www.bioinformatics.nl/emboss-explorer/">EMBOSS Explorer</a> - The EMBOSS user interface maintained by the Bioinformatics Center at the Netherlands.</li>
                        <li><a href="http://weblab.gao-lab.org/">WebLab</a> - The Sequence analysis platform developed by the Center for Bioinformatics (CBI), Peking University.</li>
                    </ol>
                    
                    <h3 id="5">Bioinformatics packages to be downloaded and installed locally</h3>
                    <ol>
                        <li><a href="http://emboss.open-bio.org/">EMBOSS</a>- The main portal for the open source bioinformatics project EMBOSS.</li>
                        <li><a href="https://www.ebi.ac.uk/interpro/search/sequence/">InterProScan</a> - Identification of protein family and domain based-on the integrated database of InterPro.</li>
                        <li><a href="http://hmmer.org/">HMMER</a>- Profile hidden Markov models to detect conserved domains in protein sequences.</li>
                        <li><a href="https://evolution.genetics.washington.edu/phylip/software.html">Phylogeny software</a>- The whole list of phylogeny programs collected and classified in groups by Joe Felsenstein.</li>
                    </ol>
                    
                    <h3 id="6">Protein sequence analysis and function prediction</h3>
                    <ol>
                        <li><a href="https://web.expasy.org/protparam/">ProtParam</a>- Computating various physical and chemical parameters for a protein sequence, including the molecular weight, theoretical pI, amino acid composition, atomic composition.</li>
                        <li><a href="https://web.expasy.org/protscale/">ProtScale</a> - Display protein sequence profiles as two-dimensional plot, including hydrophobicity, secondary structure, accessibility, flexibility, polarity, transmembrane tendency, etc.</li>
                        <li><a href="https://predictprotein.org/">PredictProtein</a> - Prediction of protein function including secondary structure, protein localization, disorder, disulfide bridges, etc.</li>
                    </ol>
                    
                    <h3 id="7">Motif finding</h3>
                    <ol>
                        <li><a href="https://weblogo.threeplusone.com/">Weblogo</a>- The web server for the generation of sequence logos, developed and maintained by University of California at Berkeley.</li>
                        <li><a href="http://smart.embl-heidelberg.de/">SMART</a> - The web server at EMBL to search known motifs in a protein sequence.</li>
                        <li><a href="https://meme-suite.org/meme/">MEME</a> - Motif-based sequence analysis tools.</li>
                        <li><a href="https://www.ebi.ac.uk/interpro/">InterPro</a> - Protein domain classification at EBI.</li>
                        <li><a href="https://services.healthtech.dtu.dk/services/SignalP-6.0/">SignalP</a> - Prediction of signal peptide at Denmark Technical University.</li>
                        <li><a href="https://services.healthtech.dtu.dk/services/TMHMM-2.0/">TMHMM</a> - Prediction of transmembrane helices at Denmark Technical University.</li>
                    </ol>
                    
                    <h3 id="8">Primer Design</h3>
                    <ol>
                        <li><a href="http://bioweb.uwlax.edu/genweb/molecular/seq_anal/primer_design/primer_design.htm">Primer Design</a> - The online primer design tool at BioWeb.</li>
                        <li><a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">Primer Blast</a> - NCBI primer design tool with Blast search.</li>
                        <li><a href="https://www.yeastgenome.org/primer3">Web Primer</a> - Online primer design tool at the yeast genome database.</li>
                    </ol>
                    
                    <h3 id="9">Gene identification</h3>
                    <ol>
                        <li><a href="http://www.softberry.com/berry.phtml/berry.phtml?topic=fgenesh&group=programs&subgroup=gfind">FGeneSh</a>- The web server for gene prediction, limited free use for academic users.</li>
                        <li><a href="http://bioinf.uni-greifswald.de/augustus/submission">AUGUSTUS</a> - Gene prediction tool at the University of Greifswald, Germany.</li>
                        <li><a href="http://hollywood.mit.edu/GENSCAN.html">GenScan</a> - The web server for the identification of complete gene structures in genomic DNA at MIT.</li>
                        <li><a href="https://services.healthtech.dtu.dk/service.php?HMMgene-1.1">HMMGene</a> - The web server for the prediction of vertebrate and C elegans genes, developed and maintained by the Denmark Technical University.</li>
                        <li><a href="http://gsds.gao-lab.org/">GSDS</a> - The Gene Structure Display Server maintained by the Center of Bioinformatics, Peking Universi.</li>
                    </ol>
                    
                    <h3 id="10">Molecular phylogeny</h3>
                    <ol>
                        <li><a href="https://www.megasoftware.net/">MEGA</a> - Molecular Evolutionary Genetics Analysis package.</li>
                        <li><a href="https://evolution.genetics.washington.edu/phylip.html">Phylip</a> - Phylogeny Inference Package created by Joe Felsenstein at the University of Washington.</li>
                        <li><a href="http://abacus.gene.ucl.ac.uk/software/paml.html">PAML</a> - Phylogenetic Analysis by Maximum Likelihood package developed and maintained by Ziheng Yang, at University College, London.</li>
                        <li><a href="http://dambe.bio.uottawa.ca/ErrPage/fileNotFound.aspx">DAMBE</a> - Data Analysis and Molecular Biology and Evolution developed by Xuehua Xia at University of Ottawa, Canada.</li>
                        <li><a href="https://itol.embl.de/">iTOL</a> - Interactive Tree of Life, for the display and manipulation of phylogenetic trees.</li>
                    </ol>
                    
                    <h3 id="11">3D visualization</h3>
                    <ol>
                        <li><a href="http://www.cgl.ucsf.edu/chimerax/">ChimeraX</a> - Interactive visualization and analysis of 3D structures at UCSF.</li>
                        <li><a href="https://spdbv.unil.ch/">Swiss-PDB Viewer</a> - A comprehensive molecular visualization and modeling package at SIB.</li>
                        <li><a href="https://pymol.org/2/">PyMol</a> - The home page of the molecular visualization system and the useful help information at PyMolWiki.</li>
                        <li><a href="http://www.ks.uiuc.edu/Research/vmd/">VMD</a> - Modeling, visualization, and analysis of biological systems such as proteins, nucleic acids, lipid bilayer assemblies.</li>
                        <li><a href="http://www.umass.edu/microbio/rasmol/">RasMol</a> - The home page of molecular visualization freeware RasMol, maintained by University of Massachusetts Amherst.</li>
                    </ol>
                    
                    <h3 id="12">Protein modeling</h3>
                    <ol>
                        <li><a href="https://swissmodel.expasy.org/">Swiss Model</a> - Automated comparative protein modeling server at SIB.</li>
                        <li><a href="http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index">Phyre</a> - Protein Homology/analogY Recognition Engine at Imrerial College, UK.</li>
                        <li><a href="http://www.swissdock.ch/">SwisDock</a> - Prediction of molecular interactions between a target protein and a small molecule.</li>
                        <li><a href="https://autodock.scripps.edu/">AutoDock</a> - The web site for protein/protein and protein/ligand docking at Scripps Institute.</li>
                        <li><a href="https://predictioncenter.org/">CASP</a> - The international protein structure prediction center.</li>
                    </ol>
                    
                    <h3 id="13">RNA tools</h3>
                    <ol>
                        <li><a href="http://www.unafold.org/">UNAFOLD</a> - The UNAFold web server for mfold & DINAMelt.</li>
                        <li><a href="https://rth.dk/resources/rain/">RAIN</a> - The web server for RNA-protein Asociation and Interaction Netwroks, developed by the Danish Center for Nocoding RNA (RTH).</li>
                        <li><a href="https://rth.dk/resources/foldalign/server/">FoldAlign</a> - The web server for RNA structure and sequence alignment, developed by the Danish Center for Noncoding RNA (RTH).</li>
                        <li><a href="https://rth.dk/resources/rnasnp/">RNASNP</a> - The web server for predicting SNP effects on local RNA secondary structure, develped by the Danish Center for Noncoding RNA (RTH).</li>
                        <li><a href="https://rth.dk/resources/rilogo/">RILogo</a> - The web server for creating RNA-RNA interaction logos for two RNAs, developed by the Danish Center for Noncoding RNA (RTH).</li>
                    </ol>
                    
                    <h3 id="14">List of bioinformatics tools at international bioinformatics centers</h3>
                    <ol>
                        <li><a href="https://www.expasy.org/">ExPASy tools</a> - A comprehensive list of online web-based bioinformatics tools provided by ExPASy and worldwide.</li>
                        <li><a href="http://bioinfo.au.tsinghua.edu.cn/software-database/">Tsinghua</a> - List of software tools developed by the Bioinformatics Division, Tsinghua University.</li>
                        <li><a href="https://services.healthtech.dtu.dk/">DTU HeathTech Services</a>- The entry page for the bioinformatics tools at the Department of Health Technology, the Technical University of Denmark.</li>
                        <li><a href="https://www.bioinformatics.org/sms2/">SMS</a> - Sequence manipulation suite at Bioinformatics.org.</li>
                        <li><a href="https://genomevolution.org/coge/">GoGe</a> - The platform for Gomparative Genomics.</li>
                        <li><a href="http://www.softberry.com/">Softberry</a> - Online tools integrated by the Softberry bioinformatics company in US.</li>
                    </ol>
                </td>
            </tr>
        </table>
        </td>
    </tr>
</table>

<div class="footer">
    <h2 class="p2">蔬菜种质创新与分子设计育种实验室<br>
        Laboratory of Vegetable Germplasm Innovation and<br>
        Molecular Design Breeding Copyright @ Tao Lab<br>
        邮编：100094
    </h2>
</div>
</body>
</html>